src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/entities/TaskResourceSummary.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.cft.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.time.OffsetDateTime;
+
+@Getter
+@AllArgsConstructor
+public class TaskResourceSummary {
+
+    private String taskId;
+    private OffsetDateTime dueDateTime;
+    private String caseId;
+    private String caseName;
+    private String caseCategory;
+    private String locationName;
+    private String title;
```

Comment:
This is probably a future improvement if there's any performance gain but I will note it here.
For each search we will know the sorting parameters from the request.  What we allow is only 1 sorting parameter and then apply the "default" sorting as a secondary (which is currently by dueDate).

With that in mind we _could_ reduce the columns we retrieve to just taskId, thePrimarySortColumn, dueDate.

Not sure if that improves the performance much but every small gain may add up.